integrations.json

```diff
@@ -468,6 +568,13 @@
           "assembly": "ServiceStack.Redis",
           "type": "ServiceStack.Redis.RedisNativeClient",
           "method": "SendReceive",
+          "signature_types": [
+            "T",
+            "_",
+            "System.Int8[][]",
```

Comment:
```suggestion
            "System.Byte[][]",
```

Reply:
Nvm this will be regenerated. I'll put the suggestion in the right place in the TargetSignatureTypes...